### PR TITLE
fix swin train param is not match bug

### PR DESCRIPTION
**Pull request by wm901115nwpu**

### CODE REVIEW

**Comment by byshiue** [comment]
Can you explain what bug do you want to fix? 

Besides, please revert the changing of vs code settings.

**Comment by wm901115nwpu** [comment]
save_checkpoint(config, epoch, model_without_ddp, max_accuracy, optimizer, lr_scheduler, logger)

lack of loss param
